### PR TITLE
Add SatelliteResourceLanguages property

### LINKED ISSUE #18995: Document the SatelliteResourceLanguages MSBuild property
In .NET Core 2.1 (SDK 2.1.300) we added support for specifying languages to preserve satellite resource assemblies for during publish. See dotnet/sdk#774 for details.

``` xml
<PropertyGroup>
  <SatelliteResourceLanguages>en-US</SatelliteResourceLanguages>
</PropertyGroup>
```

In a new .NET Core WinForms app, setting this property to "en-US" (the runtime's default language) will result in a saving of 15 MB on the size of the app when published self-contained. When combined with the `PublishTrimmed` flag, this is an ~18% size saving.

Other packages commonly used in .NET Core apps (including ASP.NET Core, WinForms, and WPF apps) include embedded satellite resource assemblies that don't provide any benefit in cases where the app is only ever run in a single language context anyway.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 59d4dafc-2217-9a7b-4590-27faa85164cd
* Version Independent ID: 64b80e80-1c15-c293-466a-918de98e96f6
* Content: [MSBuild properties for Microsoft.NET.Sdk - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props)
* Content Source: [docs/core/project-sdk/msbuild-props.md](https://github.com/dotnet/docs/blob/master/docs/core/project-sdk/msbuild-props.md)
* Product: **dotnet-core**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #18995 

[Preview link](https://review.docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props?branch=pr-en-us-26171#satelliteresourcelanguages).

### CODE REVIEW

File: docs/core/project-sdk/msbuild-props.md [hunk 1]
```diff
@@ -396,6 +397,19 @@ The `RuntimeIdentifiers` property lets you specify a semicolon-delimited list of
 </PropertyGroup>
 ```
 
+## SatelliteResourceLanguages
+
+The `SatelliteResourceLanguages` property lets you specify which languages you want to preserve satellite resource assemblies for during publish. Many NuGet packages include localized resource satellite assemblies in the main package. For projects that reference these NuGet packages that don't require localized resources, the localized assemblies can unnecessarily inflate the publish output size. By adding the `SatelliteResourceLanguages` property to your project file, only localized assemblies for the languages you specify will be included in the publish output. For example, in the following project file, only English (US) resource satellite assemblies will be retained.
```
Comment (DamianEdwards) [inline docs/core/project-sdk/msbuild-props.md:402]:
```suggestion
The `SatelliteResourceLanguages` property lets you specify which languages you want to preserve satellite resource assemblies for during build and publish. Many NuGet packages include localized resource satellite assemblies in the main package. For projects that reference these NuGet packages that don't require localized resources, the localized assemblies can unnecessarily inflate the build and publish output size. By adding the `SatelliteResourceLanguages` property to your project file, only localized assemblies for the languages you specify will be included in the build and publish output. For example, in the following project file, only English (US) resource satellite assemblies will be retained.
```

**Review by DamianEdwards — APPROVED** [review]
Looks great! The property applies equally to build and publish output so might be worth making that clear.

**Review by BillWagner — APPROVED** [review]
Thanks @gewarren 

This LGTM, once you accept Damien's suggestion.

**Comment by gewarren** [comment]
Thanks Damian and Bill!

**Pull request merged**